doc/usage.rst

```diff
@@ -121,6 +124,33 @@ fixed but without actually modifying them:
 By using ``--using-cache`` option with ``yes`` or ``no`` you can set if the caching
 mechanism should be used.
 
+The ``list-files`` command
+--------------------------
+
+The ``list-files`` command will list all files which need fixing.
+
+.. code-block:: console
+
+    $ php php-cs-fixer.phar list-files
+
+The ``--config`` option can be used, like in the ``fix``  command, to tell from which path a config file should be loaded
+
+.. code-block:: console
+
+    $ php php-cs-fixer.phar list-files --config=.php_cs.dist
+
+The output is build in a form that its easy to use in combination with ``xargs`` command in a linux pipe.
+This can be useful e.g. in situations where the caching might mechanism not available (CI, Docker) and distributing
+fixing across several processes might speedup the process.
+
+.. code-block:: console
+
+    $ php php-cs-fixer.phar list-files | xargs -n 10 -P 8 php-cs-fixer fix
```

Comment:
would it make sense to count the files and provide `files/cpu` to single job?

Reply:
to provide multiple files, you need to specify config file (ref https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/4279#issuecomment-820615223  )

Reply:
> would it make sense to count the files and provide =$files/$cpu to single job?

not sure what you mean or what you are after?

> to provide multiple files to fix command


hmm does this mean we should make the config a required argument instead of a option?